hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/response/s3/multipart/S3MultipartUploadCompleteResponse.java

```diff
@@ -125,4 +126,9 @@ protected OmKeyInfo getOmKeyInfo() {
   protected List<OmKeyInfo> getPartsUnusedList() {
     return partsUnusedList;
   }
+
+  @Override
```

Comment:
@aryangupta1998 Please remove it, as its expects LEGACY, right

Reply:
Done